MdePkg/Include/IndustryStandard/IpmiSerial.h

```diff
@@ -0,0 +1,60 @@
+/** @file

```

Comment:
I see Ipmi.h #includes all the IPMI related files.  Why is this one not included in Ipmi.h?

Reply:
Ipmi.h includes the command-related files. IpmiSerial.h is IPMI related interface, such as kcs, ssif, or serial. So I didn't include it in Ipmi.h.

Reply:
> I see Ipmi.h #includes all the IPMI related files. Why is this one not included in Ipmi.h?
Hi @mdkinney 
Ipmi.h includes those protocol layer header files. IpmiSerial.h is for the transport interface which is a separate header file similar to IpmiKcs and IpmiSsif.

Reply:
@mdkinney Could you please check this again and approve it if you have no concerns about this PR?
Thanks